docs/src/components/fields/RadioGroupOptionEditor.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import { FormControl, Checkbox } from 'react-bootstrap';
+import * as assign from 'object-assign';
+
+import { IFieldOptionEditorProps } from 'react-dynamic-formbuilder';
+
+interface IState { }
+
+
+export default class RadioGroupOptionEditor extends React.PureComponent<IFieldOptionEditorProps, IState> {
```

Comment:
If there is no state, you can use 'void'.

=> React.PureComponent<IFieldOptionEditorProps, void>